homeassistant/components/telegram_bot/__init__.py

```diff
@@ -68,6 +68,7 @@
 ATTR_VERIFY_SSL = "verify_ssl"
 ATTR_TIMEOUT = "timeout"
 ATTR_MESSAGE_TAG = "message_tag"
+ATTR_CHANNEL_POST = "channel_post"
```

Comment:
Where does this come from? Can it be imported from `telegram` instead?

Reply:
https://core.telegram.org/bots/api#getting-updates

Reply:
What I meant what that it's better to import the constant `UPDATE_CHANNEL_POST`, if that's the right one, from `telegram.constants` instead of defining it in Home Assistant.

Reply:
The code is written in the same style.

Using constants from telegram.constants is a good idea, but requires refactoring of code not related to the this feature.

Reply:
OK, we can do that improvement in a separate PR.